packages/react-native/package.json

```diff
@@ -42,6 +42,7 @@
   "peerDependencies": {
     "indy-sdk-react-native": "^0.1.16",
     "react-native-fs": "^2.18.0",
-    "react-native-get-random-values": "^1.7.0"
+    "react-native-get-random-values": "^1.7.0",
+    "react-native-bbs-signatures": "0.1.0"
```

Comment:
This should be `@animo-id/react-native-bbs-signatures` right? And the version should be `0.0.3`

Reply:
To make sure the package actually exist, maybe we can add it as a dev-dependency to the RN package.json?